[wg/ag] Update for AGWG charter 2025

iadawn opened the pull request.

iadawn committed:
First cleanup

iadawn committed:
More cleanup

iadawn committed:
Updates to AG draft charter

iadawn committed:
Fix reference to Backlog group

alastc commented on 2025/ag-wg.html:
```diff
@@ -150,111 +149,143 @@ <h1 id="title">DRAFT Accessibility Guidelines Working Group Charter</h1>
+          </ul>
+        </section>
+
+        <p>WCAG benefits hugely from extensive informative materials that help foster understanding of the requirements, why they are important and how to test them. This will be developed in line with the core recommendation but is not necessary its publication.</p>
```
```suggestion
        <p>WCAG benefits hugely from extensive informative materials that help foster understanding of the requirements, why they are important and how to test them. This will be developed in line with the core recommendation but is not necessary before its publication.</p>
```

alastc commented on 2025/ag-wg.html:
```diff
@@ -150,111 +149,143 @@ <h1 id="title">DRAFT Accessibility Guidelines Working Group Charter</h1>
-              <li>The AG WG is not currently working on the <a href="https://www.w3.org/TR/ATAG20/">Authoring Tool Accessibility Guidelines</a> or the <a href="https://www.w3.org/TR/UAAG20/">User Agent Accessibility Guidelines</a> specifications, except to maintain errata, though some content may be relevant to and included in WCAG 3.</li>
-            </ul>
-          </section>
+          <p>The working group will continue to maintain WCAG 2. The primary focus of the group is to develop a CR ready version of WCAG 3 so only the most serious normative content problems will be addressed. For example, the current handling of text presentation for non-Latin script languages means that readers of those languages with disabilities are offered no support.</p>
```
```suggestion
          <p>The working group will continue to maintain WCAG 2. Only the most serious normative content problems will be addressed because the primary focus of the group is WCAG 3. For example, the current handling of text presentation for non-Latin script languages means that readers of those languages with disabilities are offered no support.</p>
```

alastc commented on 2025/ag-wg.html:
```diff
@@ -150,111 +149,143 @@ <h1 id="title">DRAFT Accessibility Guidelines Working Group Charter</h1>
-          </section>
+          <p>The working group will continue to maintain WCAG 2. The primary focus of the group is to develop a CR ready version of WCAG 3 so only the most serious normative content problems will be addressed. For example, the current handling of text presentation for non-Latin script languages means that readers of those languages with disabilities are offered no support.</p>
 
+          <p>The working group will address <a href="https://github.com/w3c/wcag/issues">open WCAG 2 issues</a> and public feedback through the WCAG 2 Maintenance TF. The TF maintains <a href="https://www.w3.org/WAI/WCAG22/Understanding/">Understanding WCAG 2</a> and <a href="https://www.w3.org/WAI/WCAG22/Techniques/">WCAG 2 Techniques</a> as informative support materials that help with interpret ...
```
```suggestion
          <p>The working group will address <a href="https://github.com/w3c/wcag/issues">open WCAG 2 issues</a> and public feedback through the WCAG 2 TF. The TF maintains <a href="https://www.w3.org/WAI/WCAG22/Understanding/">Understanding WCAG 2</a> and <a href="https://www.w3.org/WAI/WCAG22/Techniques/">WCAG 2 Techniques</a> as informative support materials that help with interpretation and education for its normative guidance.</p>
```

alastc commented on 2025/ag-wg.html:
```diff
@@ -150,111 +149,143 @@ <h1 id="title">DRAFT Accessibility Guidelines Working Group Charter</h1>
+              <p>This specification defines the format, logic, and implementation of accessibility test rules.</p>
 
+              <p class="draft-status"><b>Draft state:</b> <a href="https://www.w3.org/TR/act-rules-format/">Candidate Recommendation</a></p>
+              <p class="milestone"><b>Expected completion:</b> Q4 2027</p>
```
```suggestion
              <p class="milestone"><b>Expected completion:</b> Q1 2026</p>
```

alastc commented on 2025/ag-wg.html:
```diff
@@ -264,19 +295,6 @@ <h3>
             <li>Primer or Best Practice documents to support web developers when designing applications.</li>
```
```suggestion
            <li>Primer or Best Practice documents to support web developers when designing applications.</li>
            <li>A mapping document of WCAG 3 to WCAG 2 requirements.</li>
```

alastc commented on 2025/ag-wg.html:
```diff
@@ -150,111 +149,143 @@ <h1 id="title">DRAFT Accessibility Guidelines Working Group Charter</h1>
       <section id="background" class="background">
         <h2>Motivation and Background</h2>
-        <p>Web accessibility means that websites, tools, and technologies are designed and developed so that people with disabilities can use them. Web accessibility encompasses all disabilities that affect access to the Web, and also benefits older users and people without disabilities. Accessible design improves overall user experience and satisfaction, especially in a variety of situations, an ...
+        <p>Accessibility means that websites, tools, and technologies are designed and developed so that people with disabilities can use them. Accessibility encompasses all disabilities that affect access to the web. This also benefits older users and people without disabilities. Accessible design improves overall user experience and satisfaction, especially in a variety of situations, and acros ...
```
```suggestion
        <p>Digital accessibility means that websites, tools, and technologies are designed and developed so that people with disabilities can use them. Accessibility encompasses all disabilities that affect access to the web. This also benefits older users and people without disabilities. Accessible design improves overall user experience and satisfaction, especially in a variety of situations, and across different devices. It is essential that several different components of web development and interaction work together in order for the web to be accessible to people with disabilities. These components include content technologies, web content, web browsers and media players, assistive technology, users, developers, authoring tools, and evaluation tools. Groups in the Web Accessibility Initiative work together to address accessibility for all the components.</p>
```

iadawn committed:
Update 2025/ag-wg.html

Co-authored-by: Alastair Campbell <ac@alastc.com>

iadawn committed:
Update 2025/ag-wg.html

Co-authored-by: Alastair Campbell <ac@alastc.com>

iadawn committed:
Update 2025/ag-wg.html

Co-authored-by: Alastair Campbell <ac@alastc.com>

iadawn committed:
Update 2025/ag-wg.html

Co-authored-by: Alastair Campbell <ac@alastc.com>

iadawn committed:
Update 2025/ag-wg.html

Co-authored-by: Alastair Campbell <ac@alastc.com>

iadawn committed:
Updates from comments

iadawn committed:
Feedback changes

hidde commented on 2025/ag-wg.html:
```diff
@@ -150,133 +148,153 @@ <h1 id="title">DRAFT Accessibility Guidelines Working Group Charter</h1>
+          <p>The group develops and maintains extensive accessibility supporting materials. During this charter period, AG plans to:</p>
+          <ul>
+            <li>Update <a href="https://www.w3.org/TR/wcag2ict/">WCAG2ICT</a> with non-normative guidance on applying WCAG 2.1 and WCAG 2.2 to non-web technology.</li>
+            <li>Develop <a href="https://www.w3.org/TR/wcag2mobile-22/">Guidance on Applying WCAG 2.2 to Mobile Applications (WCAG2Mobile)</a>.</li>
```
```suggestion
            <li>Develop <a href="https://www.w3.org/TR/wcag2mobile-22/">Guidance on Applying WCAG 2.2 to Mobile Applications (WCAG2Mobile)</a>.</li>
            <li>Develop <a href="https://w3.org/TR/WCAG-EM">W3C Accessibility Guidelines Evaluation Methodology (WCAG-EM)</a>.
```

hidde replied:
(updated just now to reflect today's AGWG discussion)

julierawe commented:
Typo in 2.4 section, 3rd bullet: "engages" should be "engage," thanks.

carriehall1001 commented:
Question: (sorry if I inserted that in the wrong place) Does "Digital Accessibility" include mobile? that's not clear to me

iadawn committed:
Update 2025/ag-wg.html

Co-authored-by: Alastair Campbell <ac@alastc.com>

iadawn committed:
Update 2025/ag-wg.html

Co-authored-by: Hidde de Vries <hidde@hiddedevries.nl>

iadawn committed:
Draft changes related to charter feedback

daniel-montalvo commented on 2025/ag-wg.html:
```diff
@@ -137,10 +134,11 @@ <h1 id="title">DRAFT Accessibility Guidelines Working Group Charter</h1>
-                    <li>Roy Ran (0.05 FTE for <a href="https://www.w3.org/WAI/GL/task-forces/coga/">COGA TF</a>)</li>
-                    <li>Daniel Montalvo (0.2 FTE for <a href="https://www.w3.org/WAI/GL/task-forces/conformance-testing/">ACT TF</a>)</li>
+                    <li>Kevin White (0.8 FTE <b>primary AG WG team contact</b>)</li>
+                    <li>Ken Franquerio (0.1 FTE for <a href="https://www.w3.org/groups/tf/wcag2x-backlog/">WCAG 2 Maintenance Task Force (TF)</a>)</li>
```
```suggestion
                    <li>Ken Franqueiro (0.1 FTE for <a href="https://www.w3.org/groups/tf/wcag2x-backlog/">WCAG 2 Maintenance Task Force (TF)</a>)</li>
```

rachaelbradley committed:
Additional proposed changes for consideration

Most changes add slightly more specificity and clearly state the scope as beyond web accessibility. Please incorporate what makes sense and delete this branch as needed.

iadawn committed:
Update 2025/ag-wg.html

iadawn committed:
Update 2025/ag-wg.html

iadawn committed:
Update 2025/ag-wg.html

iadawn committed:
Update 2025/ag-wg.html

iadawn committed:
Merge pull request #754 from rachaelbradley/proposed-additions

Additional proposed changes for consideration

iadawn committed:
Update 2025/ag-wg.html

Co-authored-by: Daniel Montalvo <49305434+daniel-montalvo@users.noreply.github.com>

iadawn commented on 2025/ag-wg.html:
```diff
@@ -150,133 +148,168 @@ <h1 id="title">DRAFT Accessibility Guidelines Working Group Charter</h1>
+            </dd>
+					</dl>
 
+          <p>Maintenance of WCAG 2 may result in the need for a new version of the normative specification. While this is not a priority it is recognised that it may be necessary to address egregious issues.</p>
```
```suggestion
          <p>Maintenance of WCAG 2 may result in the need for a new version of the normative specification. While this is not a priority it is recognized that it may be necessary to address egregious issues.</p>
```

iadawn committed:
Update 2025/ag-wg.html

iadawn committed:
Add additional informative docs

alastc commented on 2025/ag-wg.html:
```diff
@@ -150,133 +148,169 @@ <h1 id="title">DRAFT Accessibility Guidelines Working Group Charter</h1>
+        <section id="wcag2">
+          <h3>WCAG 2 Maintenance</h3>
+
+          <p>The working group will continue to maintain WCAG 2. In order to do that, AG may publish a normative update in this charter period to address issues with internationalization of WCAG 2. For example, the current handling of text presentation for non-Latin script languages means that readers of those languages with disabilities are offered no support. If a normative update is needed to  ...
```
```suggestion
          <p>The working group will continue to maintain WCAG 2. In order to do that, AG may publish a normative update in this charter period to address issues with the internationalization of WCAG 2. For example, the current handling of text presentation for non-Latin script languages means that readers of those languages with disabilities are offered no support. If a normative update is needed to resolve these challenges, then a limited number of changes to address interpretation issues may also be considered. This additional work would not include new success criteria beyond that which might be required to address known internationalization issues. Overall the work will primarily be done by the WCAG 2 task force, and whole-group meeting time on this topic would be time-boxed. For example, 30 minutes every two weeks. If an issue cannot reach consensus then the current text remains.</p>
```

iadawn committed:
Update 2025/ag-wg.html

Co-authored-by: Alastair Campbell <ac@alastc.com>
